opm/simulators/linalg/ISTLSolverEbos.hpp

```diff
@@ -313,6 +277,7 @@ class WellModelGhostLastMatrixAdapter : public Dune::AssembledLinearOperator<M,X
         typedef typename GET_PROP_TYPE(TypeTag, ThreadManager) ThreadManager;
         typedef typename GridView::template Codim<0>::Entity Element;
         typedef typename GET_PROP_TYPE(TypeTag, ElementContext) ElementContext;
+        using FlexibleSolverType = Dune::FlexibleSolver<Matrix, Vector>;
```

Comment:
This is correct, but in the near future I would like to make the FlexibleSolver take the operator type as template argument, at which point we will need to switch to storing a pointer to the abstract base class instead.